packages/dev/core/src/Rendering/boundingBoxRenderer.ts

```diff
@@ -354,7 +352,7 @@ export class BoundingBoxRenderer implements ISceneComponent {
                 engine.setDepthFunctionToLess();
             }
             this._uniformBufferFront.bindToEffect(drawWrapperFront.effect!, "BoundingBoxRenderer");
-            this._uniformBufferFront.updateDirectColor4("color", frontColor);
+            this._uniformBufferFront.updateDirectColor4("color", this.frontColor.toColor4());

```

Comment:
you should replace with: updateColor4(this.frontColor, 1); cause at the moment you are introducing a perf regression by creating a color 4 on each iteration.